textile/features.textile

```diff
@@ -1375,65 +1375,22 @@ h4. Stats
-** @(TS9c)@ @mean@ - float. The average number of resources of this type used for this period.
-** @(TS9d)@ @min@ - integer. The minimum total resources of this type used for this period.
-** @(TS9e)@ @refused@ - integer. The number of resource requests refused within this period.
-* @(TS10)@ @Stats.PushStats@ - detailed stats on push notifications. Has the following attributes:
-** @(TS10a)@ @messages@ - integer. Total number of push channel messages.
-** @(TS10b)@ @directPublishes@ - integer. Total number of direct publishes.
-** @(TS10c)@ @notifications@ - a @PushNotificationCount@. The count of push notifications.
-* @(TS13)@ @Stats.PushNotificationCount@ - Contains a count of push notifications published broken down by outcome. Has the following attributes:
-** @(TS13a)@ @invalid@ - integer. Total number of attempted push notifications which were rejected due to invalid request data.
-** @(TS13b)@ @attempted@ - integer. Total number of attempted push notifications including notifications which were rejected as invalid or failed to publish.
-** @(TS13c)@ @successful@ - integer. Total number of delivered push notifications.
-** @(TS13d)@ @failed@ - integer. Total number of refused push notifications.
-* @(TS11)@ @Stats.XchgMessages@ - contains data about usage of Ably API Streamer as a producer or consumer. Has the following attributes:
-** @(TS11a)@ @all@ - a @MessageTypes@. Data about all Ably API Streamer usage.
-** @(TS11b)@ @producerPaid@ - a @MessageDirections@. Data about Ably API Streamer usage which was charged to the producer.
-** @(TS11c)@ @consumerPaid@ - a @MessageDirections@. Data about Ably API Streamer usage which was charged to the consumer.
-* @(TS14)@ @Stats.MessageDirections@ - contains data about published and received messages. Has the following attributes:
-** @(TS14a)@ @all@ - a @MessageTypes@. Data about both received and published messages.
-** @(TS14b)@ @inbound@ - a @MessageTraffic@. Data about received messages.
-** @(TS14c)@ @outbound@ - a @MessageTraffic@. Data about published messages.
+* @(TS15)@ The attributes of a @Stats@ object consist of:
+** @(TS15a)@ @intervalId@ (property present in the JSON) - a @String@
+** @(TS15b)@ @unit@ (property present in the JSON) - a @String@ or (if idiomatic) a value of the enumerable type @StatsIntervalGranularity@ whose permitted values are @minute@, @hour@, @day@, and @month@. This must be from the @unit@ property of the JSON, not calculated from the @intervalId@
+** @(TS15c)@ @inProgress@ (property present in the JSON) - an optional @String@ containing the last sub-interval included in this entry (in format yyyy-mm-dd:hh:mm:ss) for entries that are still in progress, such as the current month:
+** @(TS15d)@ @entries@ (property present in the JSON) - a @Dict<String, int>@ containing statistics entries
+* @(TS12)@ This clause has been deleted. It was valid up to and including specification version 2.0.
```

Comment:
Looks good - would it be worth ordering TS4-14 in numeric order? It makes me slightly nervous 😆

Reply:
hahaha this is the order they were in previously, but sure I've done that 👍

Reply:
:D